static/app/utils/discover/genericDiscoverQuery.tsx

```diff
@@ -47,6 +47,7 @@ export type DiscoverQueryProps = {
    * on the OrganizationEventsV2Endpoint view.
    */
   referrer?: string;
+  miseryKey?: symbol | undefined;
```

Comment:
I'd leave it out of here unless you think miseryKey applies to every subclass / specialization of our generic discover query. Extend the type in `discoverQuery` instead (see other query specializations)